apps/meteor/app/livechat/server/api/v1/contact.ts

```diff
@@ -136,3 +141,23 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'omnichannel/contacts.search',
+	{ authRequired: true, permissionsRequired: ['view-livechat-contact'], validateParams: isGETOmnichannelContactsSearchProps },
+	{
+		async get() {
+			if (process.env.TEST_MODE?.toUpperCase() !== 'TRUE') {
+				throw new Meteor.Error('error-not-allowed', 'This endpoint is only allowed in test mode');
```

Comment:
There was a PR from pierre that abstracted this away. I think we should merge that one first and then update this PR accordignly

Reply:
Agree!